collector/netstat.go

```diff
@@ -35,10 +36,19 @@ func NewNetStatCollector() (Collector, error) {
 }
 
 func (c *netStatCollector) Update(ch chan<- prometheus.Metric) (err error) {
-	netStats, err := getNetStats()
+	netStats, err := getNetStats(procNetStat)
 	if err != nil {
 		return fmt.Errorf("couldn't get netstats: %s", err)
 	}
+	snmpStats, err := getNetStats(procSnmpStat)
+	if err != nil {
+		return fmt.Errorf("couldn't get netstats: %s", err)
```

Comment:
Forget what I said about line 68. I think the invalid value message applies to the merged stats just fine. I think it's a minor nitpick, but it could simply be "stats", but then we should start renaming the variables to just 'stats' and copying both maps to it.